content/dstu2/procedure.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Procedure | DSTU 2 API
+---
+
+# Procedure
+
+* TOC
+{:toc}
+
+## Terminology Bindings
+
+<%= terminology_table(:procedure, :dstu2) %>
+
+## Search
+
+Search for Procedures that meet supplied query parameters:
+
+    GET /Procedure?:parameters
+
+_Implementation Notes_
+
+* The [Procedure.notPerformed] modifier element is not supported and will not be returned.
+
+### Parameters
+
+ Name         | Required?                             | Type          | Description
+--------------|---------------------------------------|---------------|------------------------------------------------------------------------------------
+ `_id`        | This, or one of `patient` or `subject`| [`token`]     | The logical resource id associated with the resource. Example: `_id=24110557`
+ `patient`    | This, or one of `_id` or `subject`    | [`reference`] | The patient subject of the Procedure. Example: `patient=1316024`
+ `subject`    | This, or one of `_id` or `patient`    | [`reference`] | The subject of the Procedure. Must represent a Patient resource. May use the `:Patient` modifier. Example: `subject=Patient/1316020` or `subject:Patient=1316020`
+ `date`       | No                                    | [`date`]      | The date/time when the Procedure was performed. Must use the `ge` and/or `le` prefixes. Example: `date=le2017-02-01T10:30:00Z`
```

Comment:
Will change to N, but should note that 1) they don't all follow that pattern, and 2) it would probably be better to change all the other ones from Y/N to Yes/No